app/Http/Requests/UploadImageRequest.php

```diff
@@ -22,17 +22,16 @@ public function authorize(): bool
     public function rules(): array
     {
         return [
-            'images' => 'required',
-            'images.*' => 'image|mimes:jpeg,png,jpg,gif,svg'
+            'image' => 'required|mimes:jpeg,jpg,png,bmp,gif|max: 2000'
         ];
     }
 
     public function messages(): array
     {
         return [
```

Comment:
Png's worden niet geaccepteerd

Reply:
Maar ik denk dat we in ieder geval de grootte iets moeten ophogen, anders vallen er heel veel foto's af.